### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 2 - Esercizio 11

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#496] [Esercizi M-Z]
![11](https://github.com/user-attachments/assets/25e893b5-825b-4f9a-a247-03ff612f045e)

**Comment by matypist** [comment]
Soluzioni per le lettere $a$, $b$, $c$, $e$, $f$ ad opera della [tutor](https://github.com/sapienzastudentsnetwork/algebra/blob/main/scandiuzzi/README.md#aa-202425-scandiuzzi-tutor) Elisa **Scandiuzzi**, lezione del **8 ottobre** 2024 (`appunti_24_10_08.pdf`):

![image](https://github.com/user-attachments/assets/b22f8d26-eb91-4dc9-a816-3cde6c308c63)
